src/main/webapp/script.js

```diff
@@ -13,8 +16,11 @@ function createTable() { // eslint-disable-line no-unused-vars
     columnOne.innerText = 'RunID';
     const columnTwo = document.createElement('th');
     columnTwo.innerText = 'DataType';
+    const columnThree = document.createElement('th');
+    columnThree.innerText = 'Select';
     headerRow.appendChild(columnOne);
     headerRow.appendChild(columnTwo);
+    headerRow.appendChild(columnThree);
     table.appendChild(headerRow);
     let even = false;
     for (const key in json) {
```

Comment:
Instead of "json", you can give it a more representative name, e.g. runInformation. 
Same as other names. Always use meaningful name to avoid confusion.